src/java/apiview-java-processor/pom.xml

```diff
@@ -6,7 +6,7 @@
 
   <groupId>com.azure</groupId>
   <artifactId>apiview-java-processor</artifactId>
-  <version>1.29.0</version>
+  <version>1.29.1</version>
```

Comment:
If this follows our version strategy, I believe this should be `1.30.0` as `x.x.1` suggest there was a bug fix and I don't see anything in this PR that suggest that.

Reply:
Done.